lib/experimental/Navigation/Sidebar/User/index.stories.tsx

```diff
@@ -11,9 +11,9 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {
   args: {
-    name: "Dani Moreno",
+    firstName: "Dani",
+    lastName: "Moreno",
```

Comment:
You're not the center of the universe, Dani

Reply:
Becoming John Doe, one step at a time